index.js

```diff
@@ -55,7 +55,7 @@ class Client {
         return fn(err)
       }
 
-      if (typeof body.error === 'object') {
+      if (body.error) {
```

Comment:
This should probably be `body.error && typeof body.error === 'object'`, since `body.true.code` will throw.